modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -205,6 +205,155 @@
 					title: editor.lang.link.title,
 				});
 
+				editor.ui.addRichCombo('LineSpacing', {
```

Comment:
I think I've said it a few times, let's call this "LineHeight" :smile:

Reply:
This is weird. I'm not using a `addRichCombo` anymore. I need to squash commits for a better visualization :)